quarkchain/core.py

```diff
@@ -909,18 +909,21 @@ def __init__(
         self.signature = signature
 
     def get_hash(self):
```

Comment:
I guess you don't need the function since get_hash() from the parent will do the same thing.